analytical/tests/test_utils.py

```diff
@@ -30,11 +30,11 @@ def test_get_required_setting(self):
         # available in python >= 3.2
         if hasattr(self, 'assertRaisesRegex'):
             with self.assertRaisesRegex(AnalyticalException, "^USER_ID setting is not set$"):
-                user_id = get_required_setting("USER_ID", "\d+", "invalid USER_ID")
+                get_required_setting("USER_ID", "\d+", "invalid USER_ID")
         # available in python >= 2.7, deprecated in 3.2
         elif hasattr(self, 'assertRaisesRegexp'):
             with self.assertRaisesRegexp(AnalyticalException, "^USER_ID setting is not set$"):
-                user_id = get_required_setting("USER_ID", "\d+", "invalid USER_ID")
+                get_required_setting("USER_ID", "\d+", "invalid USER_ID")
```

Comment:
I removed the two assignments, `flake8` complained about that too. Should be fine, I guess.